packages/react-vapor/src/components/section/Section.tsx

```diff
@@ -25,7 +25,7 @@ export const Section: React.FunctionComponent<ISectionProps> = ({
     };
     const H = `h${(level || 1) + 1}`;
     return (
-        <fieldset className={classNames(className, mods, 'form-group mod-padding-children')}>
+        <fieldset className={classNames(className, mods, `level-${level || 1} form-group mod-padding-children`)}>
```

Comment:
I didn't know back then, but I think you could integrate the default value directly in the deconstruction:

```
{ 
  children, 
  ...
  level = 1
}
```

It would remove both `|| 1`